presto-hive/src/main/java/io/prestosql/plugin/hive/HiveMetadata.java

```diff
@@ -770,6 +806,29 @@ public void createTable(ConnectorSession session, ConnectorTableMetadata tableMe
             }
         });
 
+        // CSV specific properties
+        getCsvProperty(tableMetadata.getProperties(), CSV_ESCAPE)
+                .ifPresent(escape -> {
+                    if (hiveStorageFormat != HiveStorageFormat.TEXTFILE) {
```

Comment:
I think `HiveStorageFormat.CSV` is expected condition. (also, L819 and L826)

Reply:
Correct. Then it looks like tests are also missing. Thanks for catching this.